<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProfileDialogBase</class>
 <widget class="QDialog" name="ProfileDialogBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>466</width>
    <height>310</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Select Profile</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QListWidget" name="kcfg_ProfileName"/>
   </item>
   <item row="0" column="1">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="btnCreate">
       <property name="text">
        <string>&amp;Add Profile</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnDelete">
       <property name="text">
        <string>&amp;Delete Profile</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnModify">
       <property name="text">
        <string>&amp;Modify Profile</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnUpdatePassword">
       <property name="text">
        <string>&amp;Change Profile Password</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>kcfg_ProfileName</tabstop>
  <tabstop>buttonBox</tabstop>
  <tabstop>btnCreate</tabstop>
  <tabstop>btnDelete</tabstop>
  <tabstop>btnModify</tabstop>
  <tabstop>btnUpdatePassword</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ProfileDialogBase</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>272</x>
     <y>302</y>
    </hint>
    <hint type="destinationlabel">
     <x>261</x>
     <y>319</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ProfileDialogBase</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>328</x>
     <y>300</y>
    </hint>
    <hint type="destinationlabel">
     <x>347</x>
     <y>319</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
